css/src/components/form/help.scss

```diff
@@ -0,0 +1,23 @@
+$help-color: $text !default;
+$help-error-color: $danger !default;
```

Comment:
Is `danger` definitely meeting contrast reqs when paired with a medium background?